extensions/auth/portability-auth-microsoft/src/main/java/org/dataportabilityproject/auth/microsoft/MicrosoftAuthDataGenerator.java

```diff
@@ -51,7 +51,7 @@ public MicrosoftAuthDataGenerator(String redirectPath, Supplier<String> clientId
     public AuthFlowConfiguration generateConfiguration(String callbackBaseUrl, String id) {
         // constructs a request for the Microsoft Graph authorization code.
         String redirectUrl = callbackBaseUrl + redirectPath;
-        String queryPart = constructAuthQueryPart(redirectUrl, id, "user.read", "mail.read", "Contacts.ReadWrite");
+        String queryPart = constructAuthQueryPart(redirectUrl, id, "user.read", "mail.read", "Contacts.ReadWrite", "Calendars.ReadWrite");
```

Comment:
Merging in the new AuthMode can be used in determining which permissions to allow 
https://github.com/google/data-portability/commit/d8bb624c43f1ba2a24902be984f5523616f9e38e#diff-6e743d98353302c9aac7752f60ce39e2